### ISSUE TITLE
Support enum as RamlParamType

### ISSUE THREAD

**Issue by piffta** [phoenixnap/springmvc-raml-plugin#161]
When we tried to use enumeration as request param type, we got the following error:
`No enum constant com.phoenixnap.oss.ramlapisync.raml.RamlParamType.COMMONENUMTYPE`

Example:
`common-enums-lib.raml`:
```
#%RAML 1.0 Library
usage: Definition of commonly used enumerations.
types:
  CommonEnumType:
    enum: [
        "TYPE_ONE",
        "TYPE_TWO"
      ]
```
`api.raml`:
```
#%RAML 1.0
title: test-api
version: 0.0.1
baseUri: /test-api

uses:
  CommonEnumsLib: libraries/common-enums-lib.raml

/api:
  /commons/things:
    get:
      description: Get common things by common enum type
      queryParameters:
        commonType:
          type: CommonEnumsLib.CommonEnumType

```
Can you add support for using enums as request param types?

**Comment by stojsavljevic** [comment]
Supporting enums and some other (rather simple) data types as `queryParameters` will require certain amount of work.
But we will start working on this soon.

Thanks for reporting this!

**Comment by stojsavljevic** [comment]
Resolved with PR #171
